src/Contrib/Authentication/OpenIdConnect/Masa.Contrib.Authentication.OpenIdConnect.EFCore/Repositories/IdentityResourceRepository.cs

```diff
@@ -43,6 +43,7 @@ public async Task<List<IdentityResource>> GetListAsync()
         var identityResources = await _context.Set<IdentityResource>()
                                 .Include(idrs => idrs.UserClaims)
                                 .Include(idrs => idrs.Properties)
+                                .AsSplitQuery()
```

Comment:
I don't think it needs split query, you just need control left join and inner join.